minesweeper/auth/Auth.py

```diff
@@ -33,7 +35,7 @@ def sign_in(self, login, password):
             if password == 'admin':
                 return 'admin'
             else:
-                return 'Невірний пароль admin-а'
+                return self.text_messages[settings.language.lower()].auth.sign_in.invalid_admin_password
```

Comment:
Taking strings directly from the array could be tricky because every time you need to write `[setting...]`. First of all, this makes a lot of useless code, secondly, it leaves a place for errors.

The better way is to create a class, that will automatically provide the string with the right language. But I think that this implementation would require more work and maybe additional libraries.

If you want to do it simpler, you can do it this way: assign the `text_messages` variable needed translation at the start of the game.